### DISCUSSION TITLE
Develop a social media strategy for Catalyst Swarm

### DISCUSSION THREAD

**Discussion by stephen-rowan** [Catalyst-Swarm/Catalyst-Beehive#233] [Marketing]
## Catalyst Swarm - Twitter Account
https://twitter.com/CatalystSwarm

## Meeting - 4th August 2021 - 10:00 UTC
- Reddit Account ?
- Medium Account ?

## Catalyst Swarm - Youtube Channel
https://www.youtube.com/channel/UCuXdR8JMH0QriU0uOTVpg3w/videos

## Context
There is no Swarm Youtube content for Saturday, 3rd July 2021 - the GitBook looks sparse in this respect ( https://catalyst-swarm.gitbook.io/catalyst-swarm-genesis/july-2021/swarm-sessions-july-2021#saturday-3rd-july-2021 ). 
Was anything recorded ? 
Should we try to upload regularly ? 
Should we just upload short clips ?

## Related Activities

- https://github.com/Catalyst-Swarm/Catalyst-Beehive/issues/114
- https://github.com/Catalyst-Swarm/Catalyst-Beehive/issues/112

**Comment by dominiktilman** [comment]
Stephen now is an admin of the Twitter Account, too.

**Comment by stephen-rowan** [comment]
- Present draft Social Media Strategy next Monday, 9th August 2021, Core Swarm Meeting

**Comment by stephen-rowan** [comment]
## Rotate social media oversight on a weekly basis - 4th August 2021
 - Twitter, Discord, Reddit, Telegram, Youtube 
 - Draft Rota or timetable

**Comment by stephen-rowan** [comment]
## Swarm Social Media Strategy Ideation Miro Board  - 4th August 2021

- Ideation Miro Board - https://miro.com/app/board/o9J_l3or0gQ=/


- Mapping community channels
